common/http_test.cpp

```diff
@@ -1656,6 +1649,38 @@ TEST(HttpTest, TestRequestBodyReaderFailure) {
 	EXPECT_TRUE(got_read_error);
 }
 
+TEST(HttpTest, TestRequestBodyIgnored) {
```

Comment:
This is testing the server ignoring the body. We should have a similar test for the client ignoring the body.